src/translations/fr.json

```diff
@@ -294,11 +298,12 @@
     "LimitSide": "Côté",
     "LimitName": "Nom",
     "Limit_name": "Seuil",
+    "LimitNameCurrentViolation": "Nom du seuil",
     "OverloadedEquipment": "Ouvrage",
     "PermanentLimitName": "IST",
     "EquipmentOverload": "Charge",
     "LimitAcceptableDuration": "Durée",
-    "ErrFetchCurrentLimitViolationsMsg": "Une erreur est survenue lors la récupération des violations de contraintes en courant",
+    "ErrFetchViolationsMsg": "Une erreur est survenue lors la récupération des violations",
```

Comment:
"violations de contraintes" would be more usefull as a message

Reply:
Done